example/create_dependencies.py

```diff
@@ -60,10 +60,23 @@ def usage():
     """This function prints the usage information and exits. It is called if
     incorrect input parameters are supplied.
     """
-    print("{0} file1 [file2...]".format(sys.argv[0]))
+    print("{argv[0]} file1 [file2...]")
```

Comment:
You're missing an `f` on this f-string :-)

Reply:
Ha, I believe that line never worked ... `argv` doesn't exist :( I've fixed it properly now